runtime/v2/example/example.go

```diff
@@ -67,14 +80,10 @@ func NewTaskService(ctx context.Context, publisher shim.Publisher, sd shutdown.S
 type exampleTaskService struct {
 }
 
-// StartShim is a binary call that executes a new shim returning the address
-func (s *exampleTaskService) StartShim(ctx context.Context, opts shim.StartOpts) (string, error) {
-	return "", nil
-}
-
-// Cleanup is a binary call that cleans up any resources used by the shim when the service crashes
-func (s *exampleTaskService) Cleanup(ctx context.Context) (*taskAPI.DeleteResponse, error) {
-	return nil, errdefs.ErrNotImplemented
+// RegisterTTRPC allows TTRPC services to be registered with the underlying server
+func (s *exampleTaskService) RegisterTTRPC(server *ttrpc.Server) error {
```

Comment:
When I started implementing my own shim, I found it _very_ nontrivial to understand that this is required, so I suggest adding this to example shim.